Gems/ROS2/Code/Source/VehicleDynamics/VehicleInputsState.h

```diff
@@ -13,7 +13,8 @@
 namespace ROS2::VehicleDynamics
 {
```

Comment:
This file has to be reworked as we discussed

Reply:
Added specialization AZ::Vector3 and AZStd::vector<float> for InputZeroedOnTimeout.
Adjusted changes:
VehicleInput
VehicleInputDeadline